docs/reference/commandline/dockerd.md

```diff
@@ -60,6 +60,7 @@ weight = -1
       -p, --pidfile="/var/run/docker.pid"    Path to use for daemon PID file
       --raw-logs                             Full timestamps without ANSI coloring
       --registry-mirror=[]                   Preferred Docker registry mirror
+      --add-runtime=[]                       Register an additional OCI compatible runtime
```

Comment:
It might be worth adding a subsection [here](https://github.com/mlaventure/docker/blob/90a306b1cf28c1bebb4990903225e8ad4f4170d1/docs/reference/commandline/dockerd.md#docker-runtime-execution-options) to explain how this can be used, and give concrete examples. From the documentation here, I have for example no way to tell what's the right syntax for the `--add-runtime` flag?